packages/ceramic-doctype-three-id/src/three-id-doctype.ts

```diff
@@ -82,7 +85,7 @@ export class ThreeIdDoctype extends Doctype {
         }
 
         const patch = jsonpatch.compare(doctype.state.content, newContent)
-        const record: any = { owners: newOwners, content: patch, prev: doctype.head, id: doctype.state.log[0] }
+        const record: any = { metadata: newMetadata, content: patch, prev: doctype.head, id: doctype.state.log[0] }
```

Comment:
Here we should not neccessarily include a whole new metadata object. Instead we should include in the `header` only the properties of the metadata that we want to update.

Reply:
👍